third_party/bigframes_vendored/pandas/core/generic.py

```diff
@@ -1,10 +1,14 @@
 # Contains code from https://github.com/pandas-dev/pandas/blob/main/pandas/core/generic.py
 from __future__ import annotations
 
-from typing import Iterator, Literal, Optional
+from typing import Callable, Iterator, Literal, Optional, TYPE_CHECKING
 
 from bigframes import constants
 from third_party.bigframes_vendored.pandas.core import indexing
+import third_party.bigframes_vendored.pandas.core.common as common
```

Comment:
Likewise, needs a sync with `main` and update.

```suggestion
import bigframes_vendored.pandas.core.common as common
```

Reply:
done